lib/Installation/InstallationSettings/InstallationSettingsPage.pm

```diff
@@ -68,6 +68,12 @@ sub is_loaded_completely {
 
 sub access_booting_options {
     my ($self) = @_;
+    YuiRestClient::Wait::wait_until(object => sub {
```

Comment:
this will act as a sleep, that why it works, but it doesn't fix the root cause.
Every action in the libyui-rest-api acts as sleep because we cannot send to the server api request to close to each other, they need 1 second between them, so you are waiting here more time running 'any' command and works, but actually Booting content was there and we don't need to check it, the problem is with the screen not ready to receive actions.
You need to check that the screen is loaded, most likely by ensuring that software section doesn't show the message that is loading and add that logic to the general method that we use to see if it is loaded or not, otherwise this is hack and for a time due to changes in sut there is not failures, https://openqa.suse.de/tests/15859787#next_previous

Reply:
Added more general check